modes/elixir-mode/lem-elixir-mode.asd

```diff
@@ -0,0 +1,9 @@
+
+(defsystem "lem-elixir-mode"
```

Comment:
https://github.com/lem-project/lem/blob/97e322b8cd0fe31af3614516cf9cfac125587ec9/lem.asd#L122

To enable this mode, it must be added to the dependencies of this system.
Sorry it's not clear.

Reply:
Added, I put the mode at the end of the list